packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -604,11 +644,33 @@ contract Deploy is Deployer {
         console.log("DisputeGameFactory version: %s", version);
     }
 
+    function initializeSuperchainConfig() public broadcast {
+        address payable superchainConfigProxy = mustGetAddress("SuperchainConfigProxy");
+        address superchainConfig = mustGetAddress("SuperchainConfig");
+
+        bytes32 batcherHash = bytes32(uint256(uint160(cfg.batchSenderAddress())));
+
+        Proxy(superchainConfigProxy).upgradeToAndCall({
```

Comment:
This is being done thru the eoa and not thru the safe. If you feel like this is the easiest way to do it then i'm fine but it would be breaking the standard way that other things are initialized

Reply:
Fixed. The `superchainConfigProxy` is now deployed with the Safe as it's admin.